README.md

```diff
@@ -83,6 +84,9 @@ format>
 
 Where:
 
+--allow-v2-api
```

Comment:
Can we include the subject in the flag's name, e.g. Envoy? I would also include the fact that it is deprecated.

Something like: `--allow-envoy-deprecated-v2-api`.